docusaurus/docs/reference/ha/bootstrapping/certificates.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: Certificates
+sidebar_position: 20
+---
+
+# Controller Certificates
+
+For controllers to communicate and trust one another, they need certificates that have
+been generated with the correct attribute and relationships.
+
+## Requirements
+
+1. The certificates must have a shared root of trust
+2. The controller client and server certificates must contain a 
+   [SPIFFE ID](https://spiffe.io/docs/latest/spiffe-about/spiffe-concepts/#spiffe-id)
+
+## Steps to Certificate Creation
+There are many ways to set up certificates, so this will just cover a recommended configuration.
+
+The primary thing to ensure is that controllers have a shared root of trust. 
+A standard way of generating certs would be as follows:
+
+1. Create a self-signed root CA
+1. Create an intermediate signing cert for each controller
```

Comment:
```suggestion
1. Create an intermediate CA for each controller as its edge enrollment signer
```

Should we clarify that this intermediate CA must issue the controller's default identity leaves and is necessarily also the controller's edge enrollment signer? I believe that's implied by the concise phrasing here, but it could be more precise if, indeed, the edge enrollment signer _must_ be the issuer of the controller's leaves, i.e. `identity.cert` and `identity.server_cert`.

Reply:
The intermediate signing cert for the client/server certs does not need to be the same as the edge signing cert.

Reply:
Switched signing cert -> CA